docs/features/image_name_substitution.md

```diff
@@ -45,7 +45,36 @@ to:
 
 
 
+## Automatically modifying Docker Hub image names
 
+Testcontainers can be configured to modify Docker Hub image names on the fly to apply a prefix string.
+
+Consider this if:
+
+* Developers and CI machines need to use different image names. For example, developers are able to pull images from Docker Hub, but CI machines need to pull from a private registry
+* Your private registry has copies of images from Docker Hub where the names are predictable, and just adding a prefix is enough. 
+  For example, `registry.mycompany.com/mirror/mysql:8.0.22` can be derived from the original Docker Hub image name (`mysql:8.0.22`) with a consistent prefix string: `registry.mycompany.com/mirror/`
+
+In this case, image name references in code are **unchanged**.
+i.e. you would leave as-is:
+
+<!--codeinclude--> 
+[Unchanged direct Docker Hub image name](../examples/junit4/generic/src/test/java/generic/ImageNameSubstitutionTest.java) inside_block:directDockerHubReference
+<!--/codeinclude-->
+
+You can then configure Testcontainers to apply the prefix `registry.mycompany.com/mirror/` to every image that it tries to pull from Docker Hub.
+This can be done in one of two ways:
+
+* Setting environment variables `TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX=registry.mycompany.com/mirror/`
+* Via config file, setting `hub.image.name.prefix` in either:
+    * the `~/.testcontainers.properties` file in your user home directory, or
+    * a file named `testcontainers.properties` on the classpath
+    
+Testcontainers will automatically apply the prefix to every image that it pulls from Docker Hub - please verify that all [the required images](../supported_docker_environment/image_registry_rate_limiting.md#which-images-are-used-by-testcontainers) exist in your registry.
+
+Testcontainers will not apply the prefix to:
```

Comment:
```suggestion
Testcontainers will not apply the prefix to:

```

Otherwise it renders incorrectly:
<img width="803" alt="Screenshot 2020-12-08 at 3 42 43 PM" src="https://user-images.githubusercontent.com/1050762/101498151-0d75b080-396c-11eb-9922-14baffd6c257.png">

Reply:
Ah good catch, thanks